docs/sites.yml

```diff
@@ -5158,3 +5158,18 @@
   built_by: Luke Whitehouse
   built_by_url: https://twitter.com/_lukewh
   featured: false
+ - title: Mission42
+   main_url: https://mission42.zauberware.com
+   url: https://mission42.zauberware.com
+   description: >
+     Impress yourself, impress the world!
```

Comment:
Could we keep this as a description of what the site/app actually is? i.e. remove the marketing info--keep it focused if possible!

Reply:
Yes sure!